frontend/public/components/dashboard/dashboards-page/overview-dashboard/health-item.tsx

```diff
@@ -0,0 +1,246 @@
+import * as React from 'react';
+import { Map as ImmutableMap } from 'immutable';
+import {
+  DashboardsOverviewHealthOperator,
+  DashboardsOverviewHealthURLSubsystem,
+  DashboardsOverviewHealthPrometheusSubsystem,
+} from '@console/plugin-sdk';
+import HealthItem from '@console/shared/src/components/dashboard/status-card/HealthItem';
+import OperatorStatusBody, {
+  OperatorsSection,
+} from '@console/shared/src/components/dashboard/status-card/OperatorStatusBody';
+import {
+  getOperatorsHealthState,
+  getMostImportantStatuses,
+} from '@console/shared/src/components/dashboard/status-card/state-utils';
+import { HealthState } from '@console/shared/src/components/dashboard/status-card/states';
+import { FirehoseResourcesResult, AsyncComponent, resourcePath } from '../../../utils';
+import { uniqueResource } from './utils';
+import { withDashboardResources, DashboardItemProps } from '../../with-dashboard-resources';
+import { PrometheusResponse } from '../../../graphs';
+import { K8sKind } from '../../../../module/k8s';
+
+export const OperatorsPopup: React.FC<OperatorsPopupProps> = ({
+  resources,
+  operatorExtensions,
+}) => {
+  const sections = operatorExtensions
+    .map((o, index) => {
+      const operatorResources = o.properties.resources.reduce((acc, r) => {
+        acc[r.prop] = resources[uniqueResource(r, index).prop];
+        return acc;
+      }, {});
+      return (
+        <OperatorsSection
+          key={o.properties.title}
+          resources={operatorResources}
+          getOperatorsWithStatuses={o.properties.getOperatorsWithStatuses}
+          title={o.properties.title}
+          linkTo={o.properties.viewAllLink || resourcePath(o.properties.resources[0].kind)}
+          rowLoader={o.properties.operatorRowLoader}
+        />
+      );
+    })
+    .reverse();
```

Comment:
What's the intention behind reversing the `sections` array?

Reply:
we want to have CSV operators before Cluster operators thus we need to reverse the order of extensions (cluster operators are contributed by console app, CSV by operator-lifecycle). We can improve that later on if we will have any more complex cases, like including some extensions before/after others (similar to nav items) but reverse is enough for now

Reply:
Isn't it better then to explicitly sort the extensions in a way that Cluster operators come before any other ones?

Reply:
A better solution IMO is to add e.g. `priority` property to allow sorting extensions in a specific way when they are consumed.

For now, I'm OK with a simple `reverse`.